### DISCUSSION TITLE
Different questions about translating

### DISCUSSION THREAD

**Discussion by Alkmaaar** [EliteMasterEric/Teyvat.moe#271] [Q&A]
There will be (at least I hope) questions / discussions about the intricacies of translation, help with translation and various other questions regarding the genshin map. Etc..

1) https://github.com/Alkmaaar/genshinmap.github.io/blob/develop/src/data/features/mondstadt/nature/apple.json 
there are many lines with Chinese language `    "en": "Apple",
    "zh": "苹果",` as I understand it, do not touch them?

**Comment by EliteMasterEric** [comment]
I recommend you do not remove these lines. They are the Chinese translations for the features (`zh` is the language code for chinese, and `苹果` is Chinese for Apple).

Simply place the Russian translation alongside it, like so:

```
  "name": {
    "en": "Apple",
    "zh": "苹果",
    "ru": "яблоко",
    "fr": "Pomme"
  },
```

The example above provides translations for English, Chinese, Russian, and French.
